datafusion/physical-expr-common/src/aggregate/mod.rs

```diff
@@ -185,6 +185,41 @@ pub trait AggregateExpr: Send + Sync + Debug + PartialEq<dyn Any> {
     fn create_sliding_accumulator(&self) -> Result<Box<dyn Accumulator>> {
         not_impl_err!("Retractable Accumulator hasn't been implemented for {self:?} yet")
     }
+
+    /// Returns all expressions used in the [`AggregateExpr`].
+    /// First entry in the tuple corresponds to function arguments
+    /// Second entry in the tuple corresponds to order by expressions.
```

Comment:
nit: you have defined a struct for return type XD

The same to `WindowPhysicalExpressions`